setuptools_rust/build.py

```diff
@@ -46,6 +46,7 @@ class build_rust(RustCommand):
         ("debug", "d", "Force debug to true for all Rust extensions "),
         ("release", "r", "Force debug to false for all Rust extensions "),
         ("qbuild", None, "Force enable quiet option for all Rust extensions "),
+        ("locked", None, "Force enable locked option for all Rust extensions "),
```

Comment:
Given that using setup.py as an entry point is deprecated, I'm kinda 👎 on adding new flags to the `build_rust` command. (I'm not aware of any way to invoke it other than via setup.py, and I wonder if a future setuptools-rust will remove it completely to focus on extending the existing setuptools commands which build frontend expose.)

(I'd quite like to redirect anyone doing `setup.py build_rust` to `maturin`.)

Reply:
Removed the option from `build_rust` - it will still use the options if configured on each extension, though.